app/components/gh-file-uploader.js

```diff
@@ -21,7 +21,8 @@ export default Component.extend({
     labelText: 'Select or drag-and-drop a file',
     url: null,
     paramName: 'file',
-    accept: 'text/csv',
+    accept: ['text/csv'],
+    extensions: ['csv'],
```

Comment:
The main reason I went with a strings here was so that they can be expressed easily in handlebars when there are multiple accepted mime types/extensions e.g. 

```handlebars
{{gh-file-uploader
    accept="image/gif,image/jpg,image/jpeg,image/png,image/svg+xml"
    extensions="gif,jpg,jpeg,png,svg"}}
```